src/commands/top/index/processor.ts

```diff
@@ -40,6 +40,7 @@ export async function render(
     })
 
   const { author, leaderboard } = res.data
+  const filteredLeaderboard = leaderboard.filter((l: any) => l.user !== null)
```

Comment:
check cả `null` và `undefined`
```suggestion
  const filteredLeaderboard = leaderboard.filter((l: any) => l.user)
```